R/add_phylopic_tree.R

```diff
@@ -0,0 +1,84 @@
+#' Add PhyloPics to a plotted phylogenetic tree
+#'
+#' Specify existing images, taxonomic names, or PhyloPic uuids to add PhyloPic
+#' silhouettes alongside the associated leaves of a phylogenetic tree.
```

Comment:
I think somewhere in here (and maybe in the title) it would be good to clarify that this is for trees plotted in base R (as opposed to with ggtree which we support with geom_phylopic)